src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -1,40 +1,71 @@
-
 {
-    "$schema": "https://aka.ms/terminal-profiles-schema",
+  "$schema": "https://aka.ms/terminal-profiles-schema",
+
+  "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
 
-    "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+  // You can add more global application settings up here!
+  // To learn more about global settings, visit https://aka.ms/terminal-global-settings
 
-    "profiles":
+  // If enabled, selections are automatically copied to your clipboard
+  "copyOnSelect": false,
+
+  // Profiles are listed in your dropdown and they include the command line executable along with customizations
+  // To learn more about profiles, visit https://aka.ms/terminal-profile-settings
+  "profiles":
+  {
+    "defaults":
     {
-        "defaults":
-        {
-            // Put settings here that you want to apply to all profiles
-        },
-        "list":
-        [
-            {
-                // Make changes here to the powershell.exe profile
-                "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
-                "name": "Windows PowerShell",
-                "commandline": "powershell.exe",
-                "hidden": false
-            },
-            {
-                // Make changes here to the cmd.exe profile
-                "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
-                "name": "cmd",
-                "commandline": "cmd.exe",
-                "hidden": false
-            }
-        ]
+      // Put settings here that you want to apply to all profiles
     },
+    "list":
+    [
+      {
+        // Make changes here to the powershell.exe profile
+        "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+        "name": "Windows PowerShell",
+        "commandline": "powershell.exe",
+        "hidden": false
+      },
+      {
+        // Make changes here to the cmd.exe profile
+        "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
+        "name": "cmd",
+        "commandline": "cmd.exe",
+        "hidden": false
+      }
+    ]
+  },
+
+  // Add custom color schemes to this array
+  // To learn more about color schemes, visit https://aka.ms/terminal-color-schemes
+  "schemes": [],
+
+  // Add custom keybindings to this array
+  // To unbind a key combination from your defaults.json, set the command to "unbound"
+  // To learn more about keybindings, visit https://aka.ms/terminal-keybindings
+  "keybindings":
+  [
+    // Copy and paste are bound to Ctrl+Shift+C and Ctrl+Shift+V in your defaults.json
+    // These bindings additionally bind them to Ctrl+C and Ctrl+V
+    // To learn more about selection, visit https://aka.ms/terminal-selection
+    { "command": "copy", "keys": "ctrl+c" },
+    { "command": "paste", "keys": "ctrl+v" },
 
-    // Add custom color schemes to this array
-    "schemes": [],
+    // Press Ctrl+Shift+F to open the search box
+    // To learn more about selection, visit https://aka.ms/terminal-search

```

Comment:
Yeah. Not attached to this one since search is fairly straightforward. Like, if you've _ever_ used search in a text editor, you should not feel out of place. That's _good UX_